src/objects/container/container.stories.mdx

```diff
@@ -0,0 +1,50 @@
+import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import basicDemo from './demo/basic.twig';
+import './demo/styles.scss';
+import './container.scss';
+
+<Meta title="Objects/Container" decorators={[withKnobs]} />
+
+# Container
+
+<Preview>
+  <Story
+    name="Basic"
+    height="300px"
+    parameters={{ paddings: { disabled: true } }}
+  >
+    {basicDemo({
+      prose: boolean('Add o-container--prose modifier to the container', false),
```

Comment:
Could we shorten this label so it looks a bit nicer when the sidebar is aligned to the right?